mne/data/html_templates.py

```diff
@@ -59,9 +59,13 @@
 
 raw_template = Template("""
 {{info_repr[:-9]}}
+    <tr>
+        <th>Filenames</th>
+        <td>{{', '.join(filenames)}}</td>
+    </tr>
     <tr>
         <th>Measurement time range</th>
-        <td>{{u'%0.2f' % tmin}} to {{u'%0.2f' % tmax}} sec.</td>
+        <td>{{tmin}} to {{tmax}} (HH:MM:SS)</td>
```

Comment:
Just tried locally and I think this looks better
```suggestion
        <td>{{tmin}} – {{tmax}} (H:MM:SS)</td>
```